packages/backend/src/preprocessor.py

```diff
@@ -19,46 +19,48 @@
+    Answer:"""
+
+    general_prompt = PromptTemplate(
+        input_variables=["question"], template=general_prompt_template
+    )
+    in_depth_prompt = PromptTemplate(
+        input_variables=["question"], template=in_depth_prompt_template
+    )
 
-    llm_chain = LLMChain(llm=llm, prompt=prompt)
+    llm_chain_general = LLMChain(llm=llm, prompt=general_prompt)
+    llm_chain_in_depth = LLMChain(llm=llm, prompt=in_depth_prompt)
 
     base_embeddings = OpenAIEmbeddings()
 
-    embeddings = HypotheticalDocumentEmbedder(
-        llm_chain=llm_chain, base_embeddings=base_embeddings
+    general_embeddings = HypotheticalDocumentEmbedder(
+        llm_chain=llm_chain_general, base_embeddings=base_embeddings
+    )
+    in_depth_embeddings = HypotheticalDocumentEmbedder(
+        llm_chain=llm_chain_in_depth, base_embeddings=base_embeddings
     )
 
-    return embeddings
-
+    return general_embeddings, in_depth_embeddings
 
 
 def create_db_from_docx(doc_directory):
     all_docs = []
     for doc_file in os.listdir(doc_directory):
-        if not doc_file.endswith(".docx"): 
+        if not doc_file.endswith(".docx"):
             continue
         doc_path = os.path.join(doc_directory, doc_file)
         loader = Docx2txtLoader(doc_path)
         document = loader.load()
-        if not document: 
+        if not document:
             logger.error(f"No content found for Word Doc file: {doc_path}")
             continue
-        source_id = os.path.splitext(doc_file)[
-            0
-        ]  
+        source_id = os.path.splitext(doc_file)[0]
         text_splitter = RecursiveCharacterTextSplitter(
             chunk_size=2500, chunk_overlap=1250
```

Comment:
Why weren't these values also adjusted?

Reply:
Spent hours playing with agenda/minute embeddings when I initially added them. These turned out to be optimal. I suppose it's possible that they are no longer optimal now that we have different weights for different inputs. It's probably worth looking into.